wiki/UI-component/Video.md

```diff
@@ -39,6 +39,147 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 
 <hr width="40%" />
 
+## 1. Data & content
+
+The video component intakes a Kaltura `videoId` and fetches the following data to be rendered on page as content. All of these content can be customized except the video duration.
+
+See more details at [Kaltura Player API](http://player.kaltura.com/docs/api).
```

Comment:
1. Try to reference back to the resources when you call them out.
2. Use the callouts from the functional specs template for moments like these.

```suggestion
> ![image](https://user-images.githubusercontent.com/3793636/117873919-f6faba80-b265-11eb-81a5-039bdcd822e8.png)  Check out the Kaltura player <sup>[[r2](#resources)]</sup> for more information
```